config/openshift/webhook.yaml

```diff
@@ -30,23 +30,23 @@ spec:
         name: tekton-operator-webhook
         app: tekton-operator
     spec:
-      serviceAccountName: tekton-operator
+      serviceAccountName: openshift-pipelines-operator
```

Comment:
Same remark as I did to @nikhil-thomas's PR, I think this should stay `tekton-operator` as this is in `tektoncd`, OpenShift Pipelines happens to use this but we shouldn't make it hardcoded I think

Reply:
ah okay. I think that change of service account was not reverted due to which webhook is failing now.
controller uses `openshift-pipelines-operator` and webhook `tekton-operator` 
and on openshift `tekton-operator` is not created anymore

Reply:
ah I see. Then we should probably create an issue to track this 😝